core/title_loader.py

```diff
@@ -256,7 +256,7 @@ def _extract_places(self, record, title):
             # TODO fix this up so that it works with nulls
             country = _normal_place(field['a'])
             state = _normal_place(field['b'])
-            county = _normal_place(field['c'])
+            county = _normal_place(field.get('c', None))
```

Comment:
Same behavior with `county = _normal_place(field.get('c'))` as `None` is the default value if a key isn't present.